elastica/_calculus.py

```diff
@@ -9,15 +12,17 @@
 
 
 @functools.lru_cache(maxsize=2)
-def _get_zero_array(dim, ndim):
+def _get_zero_array(dim: int, ndim: int) -> Union[float, NDArray[np.float64], None]:
```

Comment:
Why is this `float` not `np.float64`?

Reply:
The function is not used anywhere and the function doesn't look useful. I'm removing it.

[901b19a](https://github.com/GazzolaLab/PyElastica/pull/367/commits/901b19ac9b736112cec1a3daccb5a364326a5489)